src/org/zaproxy/zap/view/StandardFieldsDialog.java

```diff
@@ -1159,6 +1159,43 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 		this.incTabOffset(tabIndex);
 	}
 	
+	/**
+	 * Add a 'node select' field which provides a button for showing a Node Select Dialog and a 
+	 * non editable field for showing the node selected
+	 * 
+	 * @since 2.8.0
```

Comment:
We usually add the exact version when in a release branch, until then it's just a `TODO add version`.